scripts/library.sh

```diff
@@ -84,6 +84,18 @@ function function_exists() {
   [[ "$(type -t $1)" == "function" ]]
 }
 
+# Capitalize the first letter of each word.
+# Parameters: $1..$n - words to capitalize.
+function capitalize() {
+  local words=("$1")
```

Comment:
do you need this var?

Reply:
You do, it's transforming a string into a list.